pkg/functions/client_test.go

```diff
@@ -21,6 +22,7 @@ import (
 	"time"
 
 	cloudevents "github.com/cloudevents/sdk-go/v2"
+	cmd "knative.dev/func/cmd"
```

Comment:
We only need to use the commands in the tests from within that package.  This package can test the `Client` directly